OneCommunityGlobal/HighestGoodNetworkApp
# Yili - When entering WBS management page of a project with timer on, the timer will stop

# Description
<img width="769" alt="WBS management page" src="https://github.com/user-attachments/assets/51abd320-a786-43e0-9f07-822eeeaea3d2">

## Related PRS (if any):
For backend use development branch

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. Log as admin user
5. Go to Dashboard - Other Links - Projects. 
6. Click the `WBS` button of any project. 
7. Start the timer
8. Click on the name of any WBS. If there’s none, create one and click the name of it. 
9. Verify that the timer works well when  entering the WBS management page.

## Screenshots or videos of changes:

https://github.com/user-attachments/assets/861cc1c1-b6cc-4460-805c-0fd687751ef4
